src/components/SideMenu/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Styled from './styled';
```

Comment:
Use destructuring to clean your JSX code
```suggestion
import { CloseBtn, CloseIcon, ContainerMenu, Links, LinkText, SideMenu } from './styled';
```